src/main/java/CodeCatalyst.java

```diff
@@ -0,0 +1,130 @@
+import java.util.Scanner;
+
+public class CodeCatalyst {
+    private static final int MAX_TASKS = 100;
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        Task[] tasks = new Task[MAX_TASKS];
+        int taskCount = 0;
+
+        printGreeting();
+
+        while (true) {
+            String input = scanner.nextLine();
+            printDivider();
+
```

Comment:
Main logic and flow for the chatbot could be refactored out into a separate loop function.